pom.xml

```diff
@@ -2001,15 +2001,26 @@
                 <param>com.puppycrawl.tools.checkstyle.DetailNodeTreeStringPrinter</param>
                 <param>com.puppycrawl.tools.checkstyle.AstTreeStringPrinter</param>
                 <param>com.puppycrawl.tools.checkstyle.TreeWalker</param>
-                <param>com.puppycrawl.tools.checkstyle.Main</param>
               </targetClasses>
               <targetTests>
                 <param>com.puppycrawl.tools.checkstyle.DetailNodeTreeStringPrinterTest</param>
                 <param>com.puppycrawl.tools.checkstyle.AstTreeStringPrinterTest</param>
                 <param>com.puppycrawl.tools.checkstyle.TreeWalkerTest</param>
-                <param>com.puppycrawl.tools.checkstyle.MainTest</param>
+                <param>com.puppycrawl.tools.checkstyle.checks.blocks.*</param>
```

Comment:
@Nimfadora , @rnveach 
We do testing in functional way as we care about how our Checks work the most.
It is not ideal but ok that some parts of Treewalker is covered by some Check's test. We do the same for UTs coverage, but for cobertura we do not have multiple launches.
In Pitest multiple executions is result of performance problems.
If increasing amount of test make pitest coverage better and keep execution below 10-15 min - it is ok to do (at least in our first fight with pitest coverage). We could make it better, with more specific tests later on.